### ISSUE TITLE
Configurable Swatches overwrite rounded price

### ISSUE THREAD

**Issue by gsorry** [openstream/Magento5CentRounding#6]
Configurable Swatches overwrite rounded price on product detail page by javascript.
I found this line is responsable for that in: app/design/frontend/rwd/default/template/catalog/product/view.phtml

    <script type="text/javascript">
        var optionsPrice = new Product.OptionsPrice(<?php echo $this->getJsonConfig() ?>);
    </script>

Need to fix method getJsonConfig() in Mage_Catalog_Block_Product_View

**Comment by mikek70** [comment]
Yes - you can override the formatPrice function of product.js by including this in your themes scripts.js:

`Product.OptionsPrice.prototype.formatPrice = function(price){
		 return formatCurrency(Math.round(price*20)/20, this.priceFormat);
};`
